Projects/Team3/Andrii Nazaruk/HomeWork_OOP_1_2/Program.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Serialization;
+
+namespace HomeWork_OOP_1_2
+{
+   
+    class Program
+    {
+        //"* Створити колекцію транспорту, додати кілька різних видів транспорту та автомобілів до цієї колекції
+        //реалізуйте виведення автомобілів старіших ніж 10 років
+        //відсортуйте інформацію за брендом та моделлю
+        //збережіть колекцію у файл
+        //реалізуйте опрацювання виключень(Exceptions)
+        //сереалізуйте(Serialize) колекцію в XML файл
+        //десереалізуйте колекцію з XML файла
+        //напишіть unit тести до реалізованих функцій"
+        static void Main(string[] args)
+        {   
+            //Create a list of cars
+            Car car1 = new Car(200, "X5", "BMW", "Car", 10, true, 22.277);
+            Car car2 = new Car(220, "A8", "Audi", "Car", 8, true, 35.12);
+            Car car3 = new Car(210, "A200", "Mercedes-Benz", "Car", 13, true, 24.434);
+            Car car4 = new Car(200, "Skyline", "Nissan", "Car", 11, false, 27.13);
+            Car car5 = new Car(240, "X6", "BMW", "Car", 6, true, 21.44);
+            Car car6 = new Car(200, "Omega", "Opel", "Car", 20, false, 5.4);
+            Car car7 = new Car(230, "Fusion", "Ford", "Car", 12, true, 12.3);
+            Car car8 = new Car(320, "Huricane", "Lamborgini", "Car", 3, true, 223.5);
+            Car car9 = new Car(260, "Phantom", "Rols-Royse", "Car", 5, true, 157.4);
+
+            List<Car> cars = new List<Car>();
+            cars.Add(car9);
```

Comment:
If you no need to use cars object separately from the List you can use the next construction
 cars.Add(new Car(200, "X5", "BMW", "Car", 10, true, 22.277));